great_expectations/expectations/core/expect_column_values_to_be_in_type_list.py

```diff
@@ -55,9 +55,23 @@
 
 logger = logging.getLogger(__name__)
 
+EXPECTATION_SHORT_DESCRIPTION = "Expect a column to contain values from a specified type list."
+COLUMN_ARG = "The column name."
+TYPE_LIST_ARG = """
+    A list of strings representing the data type that each column should have as entries. \
+    Valid types are defined by the current backend implementation and are dynamically loaded. \
```

Comment:
This is going to end up creating a massive tooltip in the UI. Can you shorten this variable to only hold these first 2 sentences and then keep everything from "For example" on in the docstring only?